Content/Items/Magnet/Weapons.GrayGoo.cs

```diff
@@ -0,0 +1,435 @@
+using StarlightRiver.Content.Buffs.Summon;
+using StarlightRiver.Content.Dusts;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria.DataStructures;
+using Terraria.ID;
+using static Terraria.ModLoader.ModContent;
+
+namespace StarlightRiver.Content.Items.Magnet
+{
+	public class GrayGooDustData //Has to be a class so we can pass by reference
+	{
+		public int x;
+
```

Comment:
this linebreak is probably uneccisary and having the lines defining x and y together would be more clear

Reply:
fixed